rdkit/Chem/Fingerprints/FingerprintMols.py

```diff
@@ -239,7 +238,7 @@ def FingerprintsFromDetails(details, reportFreq=10):
       colTypes = DbUtils.TypeFinder(data, len(data), len(data[0]))
       typeStrs = DbUtils.GetTypeStrings([details.idName, details.smilesName], colTypes,
                                         keyCol=details.idName)
-      cols = '%s, %s %s' % (typeStrs[0], details.fpColName, DbModule.binaryTypeName)
+      cols = f'{typeStrs[0]}, {details.fpColName}, {DbModule.binaryTypeName}'
```

Comment:
Additional comma compared to original:
```
f'{typeStrs[0]}, {details.fpColName} {DbModule.binaryTypeName}'
```

Reply:
OK, I got your idea! I would fix it in the next patch. Thank you for your review.